src/modules/auth/auth.service.ts

```diff
@@ -0,0 +1,46 @@
+import { ConflictException, HttpStatus, Injectable, UnauthorizedException } from '@nestjs/common';
+import { CustomHttpException } from 'src/shared/custom-http-filter';
+import * as bcrypt from 'bcryptjs';
+import { LoginDto } from './dto/login.dto';
+import { JwtService } from '@nestjs/jwt';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { User } from '../../entities/user.entity';
+import Utils from '../../shared/utils';
+
+@Injectable()
+export class AuthService {
+  constructor(
+    @InjectRepository(User)
+    private usersRepository: Repository<User>,
+    private jwtService: JwtService
+  ) {}
+
+  async login(loginDto: LoginDto): Promise<{}> {
```

Comment:
why are you returning a promise of empty object for login? You returns should have a shape, no?

Reply:
Issue resolved

Reply:
Still see it in the code tho